crates/iota-sdk/examples/governance_api.rs

```diff
@@ -43,14 +42,10 @@ async fn main() -> Result<(), anyhow::Error> {
 
     // List all active validators
 
-    let active_validators = match iota_system_state {
-        IotaSystemStateSummary::V1(v1) => v1.active_validators,
-        IotaSystemStateSummary::V2(v2) => v2.active_validators,
-        _ => panic!("unsupported IotaSystemStateSummary"),
-    };
+    let active_validators = iota_system_state.iter_committee_members();
 
     println!(" *** List active validators *** ");
-    active_validators.into_iter().for_each(|validator| {
+    active_validators.for_each(|validator| {
```

Comment:
rename to `committee_members` and adapt the println

Reply:
So I assume that this was changed back be `active_validators` in the end, or was this missed somehow?

Reply:
Because `let active_validators = iota_system_state.iter_committee_members();` on the above already make the `active_validators` be `Iterator`, so here we don't need to use `into_iter()` again.